search-commons/src/test/java/no/unit/nva/search2/ResourceClientTest.java

```diff
@@ -453,35 +452,10 @@ void shouldNotReturnResourcesContainingAffiliationThatShouldBeExcludedWhenAffili
             assertThat(pagedSearchResourceDto.hits(), hasSize(1));
         }
 
-        @Test
-        void shouldNotReturnResourcesContainingAffiliationThatShouldBeExcludedWhenAffiliationIsSubunitOfRequestedViewingScopeII()
-            throws BadRequestException {
-            var firstLevelOfViewingScope = URLEncoder.encode("https://api.dev.nva.aws.unit.no/cristin/organization/20754.6.0.0",
-                                                 StandardCharsets.UTF_8);
-            var secondLevelOfViewingScope = URLEncoder.encode("https://api.dev.nva.aws.unit.no/cristin/organization/20754.6.1.0",
-                                                 StandardCharsets.UTF_8);
-            var query =
-                ResourceQuery.builder()
-                    .fromQueryParameters(Map.of(ResourceParameter.TOP_LEVEL_ORGANIZATION.fieldName(), firstLevelOfViewingScope,
-                                                UNIT.fieldName(), secondLevelOfViewingScope,
-                                                EXCLUDE_SUBUNITS.fieldName(), Boolean.TRUE.toString()))
-                    .withRequiredParameters(FROM, SIZE, AGGREGATION)
-                    .withOpensearchUri(URI.create(container.getHttpHostAddress()))
-                    .build()
-                    .withRequiredStatus(PUBLISHED, PUBLISHED_METADATA, DELETED);
 
-            logger.info(query.getValue(SORT).toString());
-            var response = searchClient.doSearch(query);
-            var pagedSearchResourceDto = query.toPagedResponse(response);
-
-            var excludedSubunit = "https://api.dev.nva.aws.unit.no/cristin/organization/20754.6.1.1";
-
-            assertThat(pagedSearchResourceDto.toJsonString(), not(containsString(excludedSubunit)));
-            assertThat(pagedSearchResourceDto.hits(), hasSize(2));
-        }
```

Comment:
This test assumes that two parameters have an OR relationship, which they should never do.
adding parameters should always reduce number of hits, not add hits.